skrub/_gap_encoder.py

```diff
@@ -1044,7 +1044,7 @@ def get_kmeans_prototypes(
     projected = vectorizer.transform(X)
     if not sparse:
         projected = projected.toarray()
-    kmeans = KMeans(n_clusters=n_prototypes, random_state=random_state)
+    kmeans = KMeans(n_clusters=n_prototypes, n_init=10, random_state=random_state)
```

Comment:
In the future, the default `n_init` will change to 1. I explicitly set it to the actual default 10 to preserve the existing behavior.